src/pyOpenMS/pxds/IDFilter.pxd

```diff
@@ -19,82 +19,146 @@ cdef extern from "<OpenMS/FILTERING/ID/IDFilter.h>" namespace "OpenMS":
 
     cdef cppclass IDFilter:
```

Comment:
You could use a multiline #wrap-doc statement similar to MSExperiment.pxd and add the whole description: 
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/FILTERING/ID/IDFilter.h#L58

Reply:
Done